subprojects/packagefiles/libtommath/meson.build

```
@@ -0,0 +1,380 @@
+project(
+  'libtommath',
+  'c',
+  version: '1.3.0',
+  license: 'CC-PDDC',
```

This is not correct.
```suggestion
  license: 'Unlicense',
```